advisor/src/funTest/assets/retrieve-package-findings-expected-result.json

```diff
@@ -49,5 +49,32 @@
         "severity" : "7.5"
       } ]
     } ]
+  } ],
```

Comment:
I'm not sure if this was a misunderstanding of one of my earlier comments, but IMO this commit should be squashed with the next one, so test data and implementation are added in the same commit.

Reply:
My intention was to make the effect of the parsing obvious in the commits. Your proposal would defeat that idea. But I do actually like it and prefer to not squash. Ok?

Reply:
Makes sense, let's keep it as-is.